libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -64,6 +64,10 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
         return this
     }
 
+    override fun reset() {
+        navigator?.resetRideSession()
```

Comment:
Do we want to crash if `reset` is called before `create`? We're using `!!` across the rest of the APIs instead of `?.` 🤔